src/main/java/com/celements/mandatory/Robots_TXT.java

```diff
@@ -89,30 +92,19 @@ boolean isSkipCelementsRobots_txt() {
 
   void checkRobots_txtDocument() throws XWikiException {
     DocumentReference robotsTxtDocRef = getRobotsTxtDocRef(getContext().getDatabase());
-    XWikiDocument robotsTxtDoc;
-    if (!getContext().getWiki().exists(robotsTxtDocRef, getContext())) {
-      LOGGER.debug("Robots_txtDocument is missing that we create it. [" + getContext().getDatabase()
-          + "]");
-      robotsTxtDoc = new CreateDocumentCommand().createDocument(robotsTxtDocRef,
-          _ROBOTS_TXT_PAGE_TYPE);
-    } else {
-      robotsTxtDoc = getContext().getWiki().getDocument(robotsTxtDocRef, getContext());
-      LOGGER.trace("Robots_txtDocument already exists. [" + getContext().getDatabase() + "]");
-    }
-    if (robotsTxtDoc != null) {
-      boolean dirty = checkPageType(robotsTxtDoc);
-      dirty |= checkRobots_txt(robotsTxtDoc);
-      if (dirty) {
+    XWikiDocument robotsTxtDoc = modelAccess.getOrCreateDocument(robotsTxtDocRef);
+    boolean dirty = checkPageType(robotsTxtDoc);
+    dirty |= checkRobots_txt(robotsTxtDoc);
+    if (dirty) {
+      try {
         LOGGER.info("Robots_txtDocument updated for [" + getContext().getDatabase() + "].");
-        getContext().getWiki().saveDocument(robotsTxtDoc, "autocreate" + " HTML.robots_txt.",
-            getContext());
-      } else {
-        LOGGER.debug("Robots_txtDocument not saved. Everything uptodate. ["
-            + getContext().getDatabase() + "].");
+        modelAccess.saveDocument(robotsTxtDoc, "autocreate HTML.robots_txt.");
+      } catch (DocumentSaveException dse) {
+        throw new XWikiException(0, 0, "failed saving", dse);
       }
     } else {
-      LOGGER.trace("skip checkRobots_txt because robotsTxtDoc is null! ["
-          + getContext().getDatabase() + "]");
+      LOGGER.debug("Robots_txtDocument not saved. Everything uptodate. ["
+          + getContext().getDatabase() + "].");
```

Comment:
```suggestion
      LOGGER.debug("Robots_txtDocument not saved. Everything uptodate. [{}]",
          getContext().getDatabase());
```
why not using slf4j message formatting?